composer.json

```diff
@@ -20,5 +20,8 @@
     "php": "^7.1"
   },
   "license": "MIT",
-  "type": "prestashop-module"
+  "type": "prestashop-library",
```

Comment:
What's the difference between prestashop-module and  prestashop-library ?

Reply:
it's a fail actually i thought it was not a module but just a composer library